airbyte-cdk/python/unit_tests/sources/test_abstract_source.py

```diff
@@ -108,7 +108,7 @@ def test_discover(mocker):
     assert expected == src.discover(logger, {})
 
 
-def test_read_nonexistent_stream_raises_exception(mocker, logger):
```

Comment:
Fixture "logger" not found  [error](https://github.com/airbytehq/airbyte/runs/4799300904?check_suite_focus=true#step:5:1161)
I removed logger fixture from the tests, so they will use logger defined in the same module.